src/client/automation/playback/click/index.js

```diff
@@ -284,7 +280,7 @@ export default class ClickAutomation {
         var moveArguments = this._getMoveArguments();
 
         return this._move(moveArguments)
-            .then(() => this._mousedown())
+            .then(() => Promise.all([delay(ACTION_STEP_DELAY), this._mousedown()]))
```

Comment:
We need a comment that describes this behavior then. It's not obvious at all from code why we need it